src/modules/loginForm/LoginForm.tsx

```diff
@@ -0,0 +1,89 @@
+import {
+  Box,
+  Button,
+  Checkbox,
+  Container,
+  Flex,
+  FormControl,
+  FormErrorMessage,
+  FormLabel,
+  Heading,
+  Input,
+  Link,
+  Stack,
+} from "@chakra-ui/react";
+import React, { ReactElement } from "react";
+import { useForm } from "react-hook-form";
+
+type FormData = {
+  username: string;
+  password: string;
+};
+export default function LoginForm(): ReactElement {
+  const { register, setValue, handleSubmit, errors } = useForm<FormData>();
```

Comment:
Variable errors will be deprecated in the next major version. Please read this for more info https://react-hook-form.com/api/#errors